src/matrix_vector/ll.rs

```diff
@@ -5,7 +5,7 @@
 //! Bindings for matrix-vector functions.
 
 pub mod cblas_s {
-    use libc::{c_float, c_int};
+    use libc::{c_float};
```

Comment:
Could you run `cargo fix` and `cargo fmt` before creating you commit, this will become mandatory real soon(TM) :)